json/encode.go

```diff
@@ -679,7 +679,7 @@ func (me *mapEncoder) encode(e *encodeState, v reflect.Value, opts encOpts) {
 			e.error(&MarshalerError{v.Type(), err})
 		}
 	}
-	sort.Slice(sv, func(i, j int) bool { return sv[i].s < sv[j].s })
+	sort.Sort(byString(sv))
```

Comment:
The changes in this file are reverting back to an older version of sort - because the new Sort mechanism is only available in later version of golang (from which the original fork of this code was taken ... because it added some useful enhancements to return error field names)